src/diamond/handler/influxdbHandler.py

```diff
@@ -170,11 +174,19 @@ def _send(self):
                 else:
                     # build metrics data
                     metrics = []
-                    for path in self.batch:
-                        metrics.append({
-                            "points": self.batch[path],
-                            "name": path,
-                            "columns": ["time", "value"]})
+                    if self.influxdb_version == "0.8":
+                        for path in self.batch:
+                            metrics.append({
+                                "points": self.batch[path],
+                                "name": path,
+                                "columns": ["time", "value"]})
+                    elif self.influxdb_version == "0.9":
+                        for path in self.batch:
+                            metrics.append({
+                                "name": path,
```

Comment:
This key name should be "measurement", not "name".

This is required to be consistent with influxdb-client: https://github.com/influxdb/influxdb-python/blob/efdac997faeb36984fa4c949e1c57ca5a6e1303d/influxdb/line_protocol.py#L94